site/styles/main.scss

```diff
@@ -349,3 +349,9 @@ s72-classification-label {
 h1, h2, h3, h4, h5, h6, p, .title {
   word-wrap: break-word;
 }
+
+p,  span {
```

Comment:
Shouldn't be in this file!  This file is for overriding the core styles underneath (equivalent to local scss in shift72-template).

Reply:
`<a>` tag doesn't come directly under `p` and `span` in our scss files. Had to override this because sometimes the `span a {` or `p a {` gets autogenerated when a user adds something through admin or an input box.

Reply:
You can add `<a>` tags under `.meta-detail-synopsis`, `.page-header`, etc!

Reply:
It might still miss some things like if a link is added in the purchase modal or the bonus content section, etc. We would need to first identify each unique instance of <a> that would be used in that case.

Reply:
I think that's a better way.  Does @stajs have an opinion?

Reply:
Moreover if we just add <a> directly under `.meta-detail-synopsis, .page-header` it might also add the style if there are any buttons including the custom field btn, pricing, signup, etc

Reply:
So add more specific selectors!

Reply:
I think this could be a little dangerous leaving it here, e.g. for any links styled as buttons where you have a different hover effect than underlining the text.

Reply:
For the case where the user has added a link in Admin for the synopsis, could we not have a more specific container to limit the selector?